src/EStimLibrary/Core/ReusableIdPool.cs

```diff
@@ -210,11 +237,15 @@ public bool TryGetNextFreeId(out int id)
 
     public SortedSet<int> GetSubset(int startId, int numIds)
     {
+        if (numIds <= 0)
```

Comment:
Good case to think about: is it already covered by the way the rest of the method is carried out? Or was the GetViewBetween(x, x) giving you errors?